### PR TITLE
Proxy pipeline

### PR DESCRIPTION
**Pull request by EverLoSa**
- **Adding proxy pipe line with streaming support (Spark Streaming)**
- **Adding logging standardization for all the pipelines.**
- **UTC fix (replicated from 1.0.1)**
- **Adding pipelines modules this will make easier to add new pipelines.**
- **Support multiple instances for the same pipeline.**
- **Centralize configuration for kafka, hdfs, and hive.**

### CODE REVIEW

File: oni/utils.py [hunk 1]
```diff
@@ -1,37 +1,98 @@
...
+
+		# add handlers to logger.
+		if create_file:
+			log.addHandler(fh)
+
+		log.addHandler(ch)
+		return  log
+
+    @classmethod
+    def create_wathcher(cls,collector_path,new_file,logger):
```
Comment (natedogs911) [inline oni/utils.py]: oops, typo: should be def create_watcher

File: pipelines/dns/collector.py [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+        # set configuration.
+        self._pkt_num = self._conf['pkt_num']
+        self._pcap_split_staging = self._conf['pcap_split_staging']        
+
+        # initialize message broker client.
+        self.kafka_topic = kafka_topic
+
+        # create collector watcher
+        self._watcher =  Util.create_wathcher(self._collector_path,NewFileEvent(self),self._logger)
```
Comment (natedogs911) [inline pipelines/dns/collector.py]: type: create_watcher

File: pipelines/flow/collector.py [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+        # set configuration.
+        self._collector_path = self._conf['collector_path']
+        self._dsource = 'flow'
+        self._hdfs_root_path = "{0}/{1}".format(hdfs_app_path, self._dsource)        
+
+        # initialize message broker client.
+        self.kafka_topic = kafka_topic
+
+        # create collector watcher
+        self._watcher =  Util.create_wathcher(self._collector_path,NewFileEvent(self),self._logger)
```
Comment (natedogs911) [inline pipelines/flow/collector.py]: create_watcher

**Comment by EverLoSa** [comment]
@natedogs911 fixed

**Comment by natedogs911** [comment]
LGTM

**Pull request merged**